config/ironjetpacks/jetpacks/allthemodium.json

```diff
@@ -3,7 +3,7 @@
   "disable": false,
   "tier": 6,
   "color": "ffef0e",
-  "armorPoints": 150,
+  "armorPoints": 9,
   "enchantability": 87,
```

Comment:
Should the enchantability values be reduced as well? They're currently 87, 187 and 420 respectively, unlike the armor point values, which were reduced to keep in line with the nerf to allthemodium tools.

Reply:
fair point, probably should be

Reply:
changed to match gold, for now